addons/true-youtube-links/addon.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "True YouTube links",
+  "description": "Replaces links to the embedded site with true YT links",
+  "credits": [
+    {
+      "name": "HTML-Fan/NT"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/*"],
+      "runAtComplete": false
+    }
+  ],
+  "options": [],
+  "tags": ["community"],
+  "enabled_by_default": true
```

Comment:
This should be false

Reply:
Okay .... if you say so ...

Reply:
But why exactly? It is kinda like a fix.

Reply:
Please stop trying to discuss everything I say :( It's clearly not a fix. Links to YouTube do not go to YouTube with the intention of them not going to YouTube.

Reply:
Okay ........

Reply:
I think that we really need recommended stuff, #172